### PR TITLE
Magiclysm - Added a deep storage freezer for a single body

### PR DESCRIPTION
**Pull request by MrHrulgin**
#### Summary
Mods "Added deep storage option for a single corpse"

#### Purpose of change

There's a few larger beasts out there that are nigh-impossible to butcher where you kill them. Your current choice is to quarter them or otherwise lose parts of the corpse in the name of portability. This provides a heavy, expensive, and bulky alternative, if you want all of your inconvenience up front in the form of hauling an expensive and awkward item out into the middle of nowhere to be ready for your hunt.

#### Describe the solution

This adds an even more awkward second tier of the magical gravity freezer. It's $8000 from the trader if he deigns to carry it, is 500L in size, 160kg empty, multiplies the weight of anything inside it by 5x, and can only carry a single item. With an adult dragon corpse inside, it weighs more than 10,000 kg. The only benefits are that it has enough internal capacity to hold an adult dragon corpse, and provides a .05 multiplier to decay, allowing a corpse to stay fresh inside for over a year. When not in use as a dragon storage device, it's also helpful for other large corpses you might want to break down at your base such as owlbear or moose, assuming you have a vehicle that has cargo capacity and can handle the weight.

#### Describe alternatives you've considered

Not doing it, but I like the idea of having more options for corpse management.

#### Testing

Spawned a dragon, killed a dragon, spawned the freezer, popped the body inside. Jumped a year into the future and saw that the body was still fresh. Jumped another year into the future and saw that the body was rotten, but not yet gone.

### CODE REVIEW

**Review by Knut-Aage-Hofseth — COMMENTED** [review]
I think the idea is cool. It does not magically solve the problem, to transport a filled freezer is still difficult, if not impossible? But it allows the survivor more time to cobble together a solution.

File: data/mods/Magiclysm/items/containers.json [hunk 1]
```diff
@@ -100,5 +100,31 @@
...
+    "flags": [ "TARDIS" ],
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "holster": true,
+        "moves": 6000,
+        "max_contains_volume": "1000 L",
+        "max_contains_weight": "4850 kg",
+        "max_item_length": "999 meter",
+        "spoil_multiplier": 0.01,
```
[the remark this answers is not in the document]
Comment (MrHrulgin) [reply data/mods/Magiclysm/items/containers.json]: Whoops, thanks for catching that.

**Comment by kevingranade** [comment]
This seems less like "more options for managing the problem" and more "a easy to buy yourself out of a challenge".

**Comment by KorGgenT** [comment]
So I don't really have a problem with the idea of the item, but trade is a little easy so I'm unsure about if it would be too easy to obtain.

**Comment by Maleclypse** [comment]
Honestly, I'd much prefer a magiclysm faction camp expansion for butchering massive creatures and the ability for a vehicle part to haul non-refrigerated bodies easily similar to having the tow on a tow truck, and a more complicated refrigeration unit that weighs a ton and burns a bunch of power so that you have multiple realistic options that can get these corpses back to your home base camp expansion where the NPCs are doing most of the work for butchering.

**Comment by MrHrulgin** [comment]
@kevingranade It's already possible to buy preserved dragon components from a merchant, though at a considerable markup. This item is a middle ground between buying the components outright and setting up a field camp to do the processing on site. Less expensive than outright buying, and trading money for the time saved setting up the camp. I feel like it's a reasonable middle ground?

@KorGgenT It's got a base cost of $8k and weight of 2 in the misc items itemgroup as compared to the weight of 12 for the tier 1 freezer. with so few items being pulled from that pool each week, my expectation is that this would not show up even every month, and the player would have to be ready with $16k to afford it, in line with the cost for a ring of protection +8. I'm open to it being even rarer, more costly or spawning *very* rarely in some other area like a wizard's tower or some other location where corpse preservation would make sense. Do you have thoughts on that?

**Comment by kevingranade** [comment]
> It's already possible to buy preserved dragon components from a merchant, though at a considerable markup.

Do that then. 

> This item is a middle ground between

Middle grounds don't improve game experience, they just make everything indistinct.

**Comment by MrHrulgin** [comment]
Outside of the specific use case for dragons, it also resolves the issue of transport of other large corpses, which often don't fit in vehicles at all given the volume limit per tile. This isn't a gimmick item with one use, and it provides functionality currently lacking.

**Comment by I-am-Erk** [comment]
Although I agree with the argument against this being a solution to the described problem, freezers of this size are actually a thing, so personally I think it's more a problem of this being the use case and implementation, rather than the object itself.

In a more general sense, for large object freezing it might be more appropriate to have a cold air field generator that lets you make a deep freezer room.

**Comment by MrHrulgin** [comment]
I would love a different, mainline solution to the challenge of preserving and transporting a large corpse. For the moment, this solves a current game challenge, in a way that seems in line with mod balance regarding item cost and rarity. I'm completely open to modifications regarding attributes, rarity, or location, if there's additional feedback. 

A suggestion from Discord was that even 5x item weight was insufficient, as once it's in a vehicle weight is irrelevant. In testing, the box with a very large corpse in it weighed ~30,000 lbs, requiring a specialized vehicle with nothing else being carried to move it around while loaded. Much heavier and it would be entirely non-portable. This is a potential option, though it would relegate it entirely to use as a base freezer for large corpses. If that's the preferred direction, I'd probably want to reexamine its stats. The limit to holding only a single corpse was intended as balance so that you couldn't just toss half a dozen deer in there and drive off, but if it's going to be furniture, I might lower the internal capacity and remove the holster flag.

 The other common concern was accessibility at the trader. As mentioned above, I could make it only findable, if the current rarity is too high.

**Comment by I-am-Erk** [comment]
So, a better way to do this might be an item, like a shroud or wrapper, that can contain a single object of any size and halts decay on its contents. Essentially, a wondrous item that works like 'gentle repose' in fantasy magic. Personally, I think the coolest way to do this would be if the wrapper item itself had a decay time, so it didn't make the contents *permanently* safe but it greatly extended their lifespan. This could be something you summon by casting a spell, and/or an item that you can purchase in a sealed package to use when you need it. ("*Lucky Joe's Gentle Repose* single-use corpse wrappers")

I feel like that's way more on brand and has more gameplay use than a bespoke deep-freeze of holding

**Comment by MrHrulgin** [comment]
Interesting. If there's approval for that as a concept, I'm happy to close this and put that together instead.

**Comment by Maddremor** [comment]
How would a non-rigid item interact with a bag of holding volume reduction?

**Comment by MrHrulgin** [comment]
Looks like it handles the size reduction just fine? The numbers here are completely made up, still early in testing:
image

**Comment by MrHrulgin** [comment]
Ok, following on I-am-Erk's idea, I've built a different concept to attempt to address the core need of temporary large corpse transport and management. #72409

**Pull request closed without merging**